### PR TITLE
Add nodes for Firebase

### PR DESCRIPTION
**Pull request by zoeponta**
Added Firebase from existing iconset #118. 
And chganed directories to match Firebase category (see the image below).

firebase-menu

On my MacOS, sort of `type` is broken in api documents by using `autogen.sh`. So I fiexed it together.

### CODE REVIEW

File: scripts/generate.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ def _gen_class_name(path: str) -> str:
         return name
 
     typ_classes = {}
-    for typ, paths in typ_paths.items():
+    sorted_typ_paths = sorted(typ_paths.items(), key=lambda x:x[0])
```
Comment (mingrammer) [inline scripts/generate.py:53]: In Python 3.6 or higher, we can sort the dict by just `sorted(typ_paths.items())`. We don't need the custom key function. 

`for typ, paths in sorted(typ_paths.items()):` is enough I think.

File: config.py [hunk 2]
```diff
@@ -199,6 +201,11 @@
             "KubernetesServices": "AKS",
         },
     },
+    "firebase": {
```
Comment (mingrammer) [inline config.py:204]: Put the `firebase` key below the `gcp` key to keep the order consistency with other dict and tuples.

File: scripts/generate.py (hunk 1 above)
Comment (zoeponta) [reply scripts/generate.py]: Thank you advice. I Fixed it.

File: config.py (hunk 2 above)
Comment (zoeponta) [reply config.py]: I Fixed it.

File: config.py [hunk 3]
```diff
@@ -51,6 +52,7 @@
     "aws": ("aws", "api", "ebs", "ec2", "efs", "emr", "rds", "ml", "mq", "nat", "vpc", "waf", "sdk"),
     "azure": ("ad", "b2c", "ai", "api", "cdn", "ddos", "dns", "fxt", "hana", "hd", "id", "sap", "sql", "vm"),
     "gcp": ("gcp", "ai", "api", "cdn", "dns", "gke", "gpu", "iap", "ml", "nat", "os", "sdk", "sql", "tpu", "vpn"),
+    "firebase": ("fcm", ),
```
Comment (mingrammer) [inline config.py:55]: Last request: `Ab` of ABTesting and `Ml` of MLKit should be uppercase. Could you please add these in this tuple? (maybe `ab`, `fcm`, `ml`)

Comment (zoeponta) [reply config.py]: Just as you said, the class name is `ABTesting`, `MLKit` is correct.
I fixed it, and re-run `autogen.sh`

**Pull request merged**